updates

ctf0 opened the pull request:
- add auto-disc for laravel
- fix migration
- remove user from as its not needed
- add event for b4 saving, very importatnt incase the user have a translation system as now the revision data will be saved under the current locale instead of replacing the column value.
- add support to fillable, guarded, hidden
- dont save model `['created_at', 'updated_at', 'deleted_at’]` by default as revisionable already do that from its part
- update readme

ctf0 committed:
add auto-disc for laravel, fix migration, update readme

johannesschobel commented:
Hey @ctf0 , thank you very much for your PR.. I hope i manage to merge it tomorrow..

ctf0 committed:
add event b4 rollingback

- remove user from as its not needed
- add event for b4 saving, very importatnt incase the user have a translation system as now the revision data will be saved under the current locale instead of replacing the column value.

ctf0 committed:
update

- add support to fillable, guarded, hidden
- dont save model `['created_at', 'updated_at', 'deleted_at’]` by default as revisionable already do that from its part
